src/main/kotlin/com/goldenraven/tatooine/Application.kt

```diff
@@ -23,10 +23,12 @@ fun Application.module(testing: Boolean = false) {
     println("Wallet is already synced: $alreadySynced")
 
     val apiPassword: String = environment.config.property("wallet.apiPassword").getString()
-    val mnemonicPhrase: String = environment.config.property("wallet.mnemonic").getString()
-
+    val descriptor = environment.config.property("wallet.descriptor").getString()
+    val change_descriptor = environment.config.property("wallet.change_descriptor").getString()
+    
     val tatooineWallet: TatooineWallet = TatooineWallet
-    tatooineWallet.initializeWallet()
+    tatooineWallet.initializeWallet(descriptor,change_descriptor)
```

Comment:
Line 29 should use recommended style and add parens at the end of TatooineWallet like so:
```kotlin
val tatooineWallet: TatooineWallet = TatooineWallet()
```

Reply:
I am getting error if I write in this way, `The function invoke() is not found`

Reply:
Oh yes sorry that's because `TatooineWallet` is an `object` (singleton). You can keep the code it as is! I didn't notice initially. Thanks!